test cases/java/5 includedirs/meson.build

```diff
@@ -0,0 +1,15 @@
+# The Ninja backend used to try and pass -sourcepath repeatedly for
+# multiple includes which would discard prior includes. Since this
+# won't compile without the '.' include, this ensures that multiple
+# paths are passed in a [semi-]colon separated list instead...
+
+project('includedirsjava', 'java')
+
+javaprog = jar('myprog',
+ 'com/mesonbuild/Simple.java',
+ 'com/mesonbuild/TextPrinter.java',
+  main_class : 'com.mesonbuild.Simple',
+  include_directories : [ include_directories('com'),
+                          include_directories('.'),
```

Comment:
Since this PR adds period to the path by default, this should not be required here, right?